### ISSUE TITLE
"Thιβtre" with fix_encoding doesn't work

### ISSUE THREAD

**Issue by mde-pach** [LuminosoInsight/python-ftfy#91]
I don't know if this case is the same that #88 but recently "Thιβtre" not being fixed by ftfy (event with the master branch and with the "Nicol\u00c3\u00a1s" working)

Same question as the named author, am I using ftfy correctly or there is another problem than those fixed today ?

```python
>>> import ftfy
>>> ftfy.fix_encoding("Nicol\u00c3\u00a1s")
'Nicolás'
>>> ftfy.fix_encoding("Thιβtre")
'Thιβtre'
>>> 
```

**Comment by rspeer** [comment]
I don't think that one's a mojibake pattern that ftfy fixes.

Assuming it's supposed to say "Théâtre", then the reason it can't be auto-fixed by ftfy is #18.

**Comment by mde-pach** [comment]
Well,
read and understood. I'll try to manage this by another way.
